Code/irc/dict_puzzle.py

```diff
@@ -34,14 +32,12 @@ def get_puzzle():
 
 def get_anagram(maxlen=6):
     dict_words = [
-        word for word in get_wordlist() if len(word) > 2 and len(word) <= maxlen + 1
+        word for word in gen_wordlist() if len(word) > 2 and len(word) < maxlen + 1
     ]
     word = random.choice(dict_words)
-    while True:
-        anagram = "".join(random.sample(word, len(word)))
-        if anagram != word:
-            break
-    puzzle = "Unscramble the following word: '" + anagram + "'"
```

Comment:
what happened to "puzzle = ..." ?

Reply:
looks like it got renamed to `challenge_text`